libtransmission/session.cc

```diff
@@ -2379,7 +2379,7 @@ static void loadBlocklists(tr_session* session)
                 }
             }
             else if (auto const pathinfo = tr_sys_path_get_info(path);
-                     path && pathinfo->last_modified_at >= bininfo->last_modified_at)
```

Comment:
nit: something is wrong with indent here

Reply:
I think this is clang-format's way of wrapping an `if (a; b)` clause that would otherwise be too wide